src/script/credits.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
```

Comment:
Please use 
`#!/usr/bin/env bash`
That will allow to find bash on othre platforms as well.

Reply:
Good catch, thanks. Done